Core/source/platform/util.ts

```diff
@@ -30,3 +31,80 @@ export const setGlobals = () => {
 export const iso2022jpToUtf = (content: Buf) => {
   return dereq_encoding_japanese.convert(content, { to: 'UTF8', from: 'JIS', type: 'string' });
 }
+
+/**
+ * Create hex string from a binary
+ * @param {String} str String to convert
+ * @returns {String} String containing the hexadecimal values
+ */
+export const str_to_hex = (str: string): string => {
```

Comment:
Is this from OpenPGP.js v4? Should mark each method that is largely from elsewhere with a comment and a link to where can the original be found on GitHub.

Reply:
Yes, it is from openpgp v4. I'll mark it.